.circleci/deploy-to-pantheon.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
I think automatically creating a multidev from a PR is a important part of an agencies workflow. 

- I don't care about my code being in a multidev environment until it works and that won't happen until I open a PR
- I want as many things to be automated as possible. I don't want to have to create a multidev and I don't want to have to ask a developer reviewing my code to do the same

In my little experience with Pantheon developing one site with an agency there was not one time we created manually a multidev when we could open a PR when the code was ready for review.